pool/app/duplicate_contacts/entity.ml

```diff
@@ -0,0 +1,98 @@
+module Field = Pool_message.Field
+
+(* The script will determine a score between 0 and 1 based on how equal two
+   users are. The alert_threshold is the value from which it will be seen as a
+   possible duplicate *)
+
+let alert_threshold = 0.7
```

Comment:
Should this be customizable by tenant?

Reply:
As discussed: We will see if this is necessary